sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -261,27 +261,25 @@ export class Application extends RecordDataModel {
 }
 
 /**
- * Interface for assessment payload.
- *
- * This interface is to provide contract for the assessment payload
- * which is stored to database by workflow.
- * It is possible that more properties can be added to the assessment payload
- * without updating this interface and displayed in NOA form.
- * Whenever there is a source code update, please ensure that properties in this interface are in sync with
- * assessment payload created by camunda workflow.
+ * Interface for BaseAssessment values that are shared between FullTime and PartTime
  */
-export interface Assessment {
+export interface BaseAssessment {
   weeks: number;
+  tuitionCost: number;
+  childcareCost: number;
```

Comment:
is it `childCareCost`?

Reply:
Childcare is a single word, I guess. Also this was already implemented in the old assessment interface, I was reusing it. Please tell me if I need to change?